resources/views/welcome.blade.php

```diff
@@ -8,8 +8,20 @@
     <!-- CSRF Token -->
 
     <meta name="csrf-token" content="{{ csrf_token() }}">
-    <meta name="description" content="La première application dédiée aux professionnels de santé à la recherche d’un lit de réanimation pour leurs malades COVID+. Pour les professionnels, par les professionnels.">
+    <meta name="description" content="La première application dédiée aux professionnels de santé à la recherche d’un lit de réanimation pour leurs malades COVID+.">
     <title>{{ config('app.name', 'Laravel') }} - application de recherche de lit de réanimation pour les professionnels</title>
+    <script type="application/ld+json">
+    {
+        "@context" : "https://schema.org",
+        "@type" : "Organization",
+        "name" : "",
+        "url" : "http://www.your-site.com",
```

Comment:
à corriger pour https://covid-moi-un-lit.com/ non ?

Reply:
Yes for sure, need some rest, new PR soon